flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/AsyncCheckpointRunnable.java

```diff
@@ -129,12 +129,10 @@ public void run() {
 					checkpointMetaData.getCheckpointId());
 			}
 		} catch (Exception e) {
-			if (LOG.isDebugEnabled()) {
-				LOG.debug("{} - asynchronous part of checkpoint {} could not be completed.",
-					taskName,
-					checkpointMetaData.getCheckpointId(),
-					e);
-			}
+			LOG.info("{} - asynchronous part of checkpoint {} could not be completed.",
```

Comment:
Won't this flood log with exceptions in some cases?

There was some [relevant discussion](https://github.com/apache/flink/pull/9873/files/d62e6daf023f440f25fa2ea5a55081e513a5fcd0#diff-9437cb4899bb946f234c272be6724770) about this when it was being introduced.

CC @klion26 ?

Reply:
Yes, this was `debug` level because there would be flood log if use `INFO` or above log level. Because we support very frequent checkpoints.

Reply:
But will be reported only in exceptional situations, do they happen that frequently?

Reply:
@rkhachatryan I think in production it would not happen(I assume that users always configure checkpoint interval with minutes), but this may happen if we configure a very small checkpoint interval.

Reply:
Even with a small checkpoint interval, failure frequency should be higher, otherwise, the job will fail.
So I don't think it poses any problem.
WDYT?

Reply:
I would also be surprised if it would fail frequently on a regular basis.

On the other hand: wouldn't this be forwarded to the checkpoint coordinator and reported there anyway?

Reply:
Yes, it will be forwarded in most cases, but this forward RPC can also fail (I think it can be helpful to have it in both logs).

Reply:
Honestly, I haven't had that happen to me. Two more things we may need to be aware of are that 1)we only count `CHECKPOINT_DECLINED` and `CHECKPOINT_EXPIRED` in CheckpointFailureManager, 2) we have `notifyCheckpointAborted` now which will cancel the snapshot of tasks if one checkpoint can’t complete.

Reply:
Honestly, I haven't had that to me. I agree that in most cases this would not happen(as my last reply said), I’m not against this change, but this may happen when we set checkpoint interval to very small(maybe in test).   
1) we only count `CHECKPOINT_DECLINED` and `CHECKPOINT_EXPIRED` in `CheckpointFailureManager`, 2) we'll abort snapshot through `notifyCheckpointAbort` if some checkpoint can't complete

Reply:
I think everything said above about failure frequency is also true for expirations (in fact, there is only one counter for all types of failures).

Reply:
> Honestly, I have not had this to me, I’m not against this change here.

Ok, in that case let's not overthink it and let's try this out :) Thanks for your inputs @NicoK and @klion26 .